### ISSUE TITLE
Bug? Unhandled match case on $driver in Laravel 9

### ISSUE THREAD

**Issue by jeffreyvanhees** [laravel-notification-channels/twilio#134]
Versions: Laravel 9.0, latest version of this package.

When sending a notification I git the "Unhandled match case"-error. It occurred in  
`Illuminate / Notifications / RoutesNotifications.php` on line 46.

It's trying to match the driver but it was not found:

```
public function routeNotificationFor($driver, $notification = null)
    {
        if (method_exists($this, $method = 'routeNotificationFor'.Str::studly($driver))) {
            return $this->{$method}($notification);
        }

       // Error occurred here:
        return match ($driver) {
            'database' => $this->notifications(),
            'mail' => $this->email,
        };
    }
```

I did everything according to the documentation. The notifiable model contains a `routeNotificationForTwilio()`, but `$method = 'routeNotificationFor'.Str::studly($driver)` returns  `"routeNotificationForNotificationChannels\Twilio\TwilioChannel"` and I believe that's why the driver cannot be found.

**Comment by jeffreyvanhees** [comment]
After upgrading from 9.0 -> 9.4 it works again. Perhaps this was already known to you, but it might be good to take this into account in the minimal dependencies. :)

Thanks for the package!

**Comment by onlime** [comment]
I have just released [v4.0.0](https://github.com/laravel-notification-channels/twilio/releases/tag/4.0.0) which only supports L11+
